plugins/services/src/js/reducers/serviceForm/Volumes.js

```diff
@@ -75,8 +75,11 @@ module.exports = {
             break;
         }
 
-        return [].concat(this.externalVolumes,
-          this.localVolumes.filter(filterHostVolumes.bind(this)).map(mapLocalVolumes));
+        return [].concat(
```

Comment:
I had to change order here to match the form, otherwise errors came in improper order